packages/ta-lib/build.sh

```diff
@@ -2,14 +2,16 @@ TERMUX_PKG_HOMEPAGE=https://sourceforge.net/projects/ta-lib/
 TERMUX_PKG_DESCRIPTION="Technical analysis library with indicators like ADX"
 TERMUX_PKG_LICENSE="BSD"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.4.0
-TERMUX_PKG_REVISION=1
-TERMUX_PKG_SRCURL=https://prdownloads.sourceforge.net/ta-lib/ta-lib-$TERMUX_PKG_VERSION-src.tar.gz
-TERMUX_PKG_SHA256=9ff41efcb1c011a4b4b6dfc91610b06e39b1d7973ed5d4dee55029a0ac4dc651
+TERMUX_PKG_VERSION=0.5.0
+TERMUX_PKG_SRCURL=https://github.com/TA-Lib/ta-lib/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=43e3761cf6bc4a5ab6c675268a09a72ea074643c6e06defe5e4b4e51eae1ea50
+TERMUX_PKG_AUTO_UPDATE=false
 TERMUX_PKG_BREAKS="talib"
 TERMUX_PKG_REPLACES="talib"
 TERMUX_PKG_BUILD_IN_SRC=true
 
 termux_step_pre_configure() {
 	curl https://sourceforge.net/p/ta-lib/code/HEAD/tree/tags/release-${TERMUX_PKG_VERSION//./-}/ta-lib/LICENSE.TXT?format=raw -o $TERMUX_PKG_SRCDIR/LICENSE
```

Comment:
Correct. This is currently the LICENSE file content in the artifact. https://github.com/TA-Lib/ta-lib/issues/35
```
<!DOCTYPE html>

<html lang="en" class="no-js">
<head>
<meta content="text/html; charset=UTF-8" http-equiv="content-type" />
<title>Page not found
    - SourceForge.net
</title>
<meta name="viewport" content="width=1100">
<link rel="icon" sizes="180x180" href="https://a.fsdn.com/con/img/sandiego/logo-180x180.png" type="image/png">
<link rel="icon" sizes="any" href="https://a.fsdn.com/con/img/sandiego/svg/originals/sf-icon-orange-no_sf.svg" type="image/svg+xml">
<link rel="apple-touch-icon" sizes="180x180" href="https://a.fsdn.com/con/img/sandiego/logo-180x180.png">
<link rel="mask-icon" href="https://a.fsdn.com/con/img/sandiego/svg/originals/sf-icon-orange-no_sf.svg" color="#FF6600">
<link rel="stylesheet" href="https://a.fsdn.com/con/css/lato.css?">
<script>
if (!window.SF) { window.SF = {}; }
SF.sandiego = false;
SF.sandiego_chrome = true;
SF.cdn = "https://a.fsdn.com/con";
SF.fpid = '372aff97-076d-4720-beca-f6314b76b0bb';
</script>
...
```

Reply:
I think I've mentioned it before, this seems to be a standalone library with no apparent dependents.
We might just want to drop it.

Reply:
It was packaged because some user asked for it, but with no explanations about reasons why we should package standalone library. 
https://github.com/termux/termux-packages/issues/5522

Reply:
I'm pretty sure the package predates the "no standalone libraries without dependents" policy.

Reply:
I am fine to leave it in since other distro also package it.
https://repology.org/project/ta-lib/versions
Please fix the LICENSE file before merge.

Reply:
Got confirmation it's still being used.
- #20429